sample-app/screens/DataConfig.tsx

```diff
@@ -229,3 +230,88 @@ export var SettingsData = {
      }
   ]
 }
+
+export enum HomeListType {
```

Comment:
for consistency, can we keep the same naming a in Android SDK:
```
export enum DemoFeature {
  Payment,
  PreAuth,
  CreateCardToken,
  CheckCard,
  SaveCard,
  GooglePayPayment,
  GooglePayPreAuth,
  ApplePayPayment,
  ApplePayPreAuth,
  PaymentMethods,
  PreAuthPaymentMethods
}
```
there's no `ideal` option for now
and let's use `PascaCase` for enums https://www.typescriptlang.org/docs/handbook/enums.html

Reply:
ok, I will change all enums in the project